sorting/insertionSort.test.js

```diff
@@ -0,0 +1,26 @@
+const insertionSort = require("./insertionSort");
```

Comment:
was it possible to have a single test file for each sort and use the same tests for each sort? Maybe by injecting the sort type in the function.